recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -190,19 +197,22 @@ def generate(self):
         tc.variables["WITH_STL"] = self.options.with_stl
         tc.variables["WITH_GSL"] = self.options.with_gsl
         tc.variables["WITH_ABSEIL"] = self.options.with_abseil
-        tc.variables["WITH_OTLP"] = self.options.with_otlp
```

Comment:
You could use `self.options.get_safe("with_otlp", False)` instead of removing. In the worst case, Cmake will only generate a warning about a non-existing option, which does affect nothing.

Reply:
No, they made that a fatal error:

if(DEFINED WITH_OTLP)
  message(
    FATAL_ERROR
      "WITH_OTLP is deprecated. Please set either WITH_OTLP_GRPC=ON, WITH_OTLP_HTTP=ON, or both to ON."
  )
endif()

Reply:
nice catch 👻